src/components/Modal/Modal.vue

```diff
@@ -725,8 +725,8 @@ export default {
 	// Sizing
 	&--small {
 		.modal-container {
-			width: 390px;
-			max-width: 95%;
+			width: 400px;
+			max-width: 90%;
```

Comment:
Changing 390px to 400px is just for the sake of using a x00 number here instead of a random feeling number. Changing 95% to 90% is in order to align with the other used max values of mostly 90%.